apps/files/src/components/FilesApp.vue

```diff
@@ -1,9 +1,9 @@
   <template>
     <div id="files" class="uk-flex uk-flex-column">
-      <files-app-bar />
-      <upload-progress v-show="inProgress.length" class="uk-padding-small uk-background-muted" />
+      <files-app-bar @upload-start="onUploadStart" @upload-end="onUploadEnd" />
```

Comment:
indeed, it starts to feel like we need some event bus to globally detect such events like uploads

Reply:
On it.

Reply:
Event bus established here: https://github.com/owncloud/phoenix/pull/2542/files#diff-c900a0e34358d44a55381e5e63de50caR70

In use for example here: https://github.com/owncloud/phoenix/pull/2542/files#diff-eec759a01d0f06a12081c5e8285c0d85R68